android/java/org/chromium/chrome/browser/brave_news/BraveNewsUtils.java

```diff
@@ -103,4 +125,253 @@ public static void logFeedItem(FeedItemsCard items, String id) {
             }
         }
     }
+
+    public static boolean shouldDisplayNews() {
+        return BravePrefServiceBridge.getInstance().getShowNews()
+                && BravePrefServiceBridge.getInstance().getNewsOptIn();
+    }
+
+    public static void setChannelIcons() {
+        mChannelIcons.put("Brave", R.drawable.ic_channel_brave);
```

Comment:
are we exploring different options to get channel icons instead of hard coding it ? i think we are doing the same on desktop but this way, for new channels, we need to push new changes.

Reply:
yes, we are doing same as desktop. I have discussed with @petemill regarding this. I think we are planning to change this instead of hard coding but not sure about roadmap